plugin/trino-redshift/src/test/java/io/trino/plugin/redshift/TestRedshiftTypeMapping.java

```diff
@@ -831,7 +831,7 @@ private void runTestCases(String tableName, List<TestCase> testCases)
                             .collect(joining("), (", "VALUES (", ")")));
         }
         finally {
-            getTrinoExecutor().execute("DROP TABLE " + tableName);
+            getTrinoExecutor().execute("DROP TABLE IF EXISTS " + tableName);
```

Comment:
The table should always exist when reaching this line. Adding `IF EXISTS` option doesn't make sense for me.

Reply:
but it's not harmful either

Reply:
this is a wrong change. If CREATE TABLE succeeded, then we know that the table exists.
Why did we merge this?